src/processGraphqlRequest.ts

```diff
@@ -33,7 +39,7 @@ export default async function processGraphqlRequest(req, {
         }
         try {
             jsonBody = await processFileUploads(req)
-        } catch (e) {
+        } catch (e: any) {
```

Comment:
e: Error

Reply:
That's not allowed
https://fettblog.eu/typescript-typing-catch-clauses/